styles.css

```diff
@@ -0,0 +1,397 @@
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  max-width: 1300px;
+  margin: 0 auto;
+  background-color: #f5f8fa;
+}
+
+.grid {
+  display: grid;
+  grid-template: "left-bar main right-bar";
+  grid-template-columns: 280px 650px 350px;
+  /* border: 1px dashed black; */
+  margin-top: 100px;
+}
+
+/* Left-bar */
+.left-bar {
+  grid-area: left-bar;
+  background-color: #f5f8fa;
+
+  /* Make a display grid within left-bar
+  Rows - will determine how wide the rows are, for example, for option nav needs to be greater than tweet grid  */
+  display: grid;
+  grid-template-rows: 50px 600px 150px;
+  grid-template-areas:
+    "logo-grid-leftbar"
+    "options-nav"
+    "tweet-grid";
+}
+
+.logo-grid-leftbar {
+  grid-area: logo-grid-leftbar;
+  width: 30px;
+  margin-top: 20px;
+  margin-left: 70px;
+}
+
+.logo-grid-leftbar:hover {
+  cursor: pointer;
+}
+
+.options-nav {
+  grid-area: options-nav;
+  /* border: 1px dashed black; */
+
+  /* Make a grid within options-nav for the options */
+
+  display: grid;
+  grid-template-rows: repeat (80px 8);
+  grid-template-areas:
+    "home"
+    "explore"
+    "notifications"
+    "messages"
+    "bookmarks"
+    "lists"
+    "profile"
+    "more";
+}
+
+/* Options on left-bar */
+.home {
+  grid-area: home;
+  text-align: left;
+  margin-left: 70px;
+}
+
+.home:hover {
+  cursor: pointer;
+}
+
+.explore {
+  grid-area: explore;
+  text-align: left;
+  margin-left: 70px;
+}
+
+.explore:hover {
+  cursor: pointer;
+}
```

Comment:
At the moment this changes your cursor if you hover over anywhre on the entire left column. If you added a second class, such as class: "nav-button" to your individual buttons it would enable you to then set the cursor to only change when you mouse over the buttons themselves.

Reply:
Thank you - Will add a class so the cursor only changes when I move to only the individual buttons.